script/whitelist/delegationFaucet/DelegationFaucet.sol

```diff
@@ -134,34 +134,34 @@ contract DelegationFaucet is IDelegationFaucet, Ownable {
         uint256[] calldata shares,
         address withdrawer
     ) public onlyOwner returns (bytes memory) {
-        bytes memory data = abi.encodeWithSelector(
-            IStrategyManager.queueWithdrawal.selector,
-            strategyIndexes,
-            strategies,
-            shares,
-            withdrawer
-        );
-        return Staker(staker).callAddress(address(strategyManager), data);
+        // bytes memory data = abi.encodeWithSelector(
+        //     IStrategyManager.queueWithdrawal.selector,
```

Comment:
should we uncomment this out before merging? seems like switching to IDelegationMananger and adjusting the params

Reply:
good point, I'll adjust this

Reply:
fixed in this commit https://github.com/Layr-Labs/eigenlayer-contracts/pull/222/commits/56160b7c022ac7c6d7dbc0b4170add6f39b00620